JointPositions joints
AchievedGoal ag
Velocities vels